client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -324,8 +327,9 @@ const PluginsBrowser = ( {
 				title={ translate( 'Plugins you need to get your projects done' ) }
 				searchTerms={ [ 'seo', 'pay', 'booking', 'ecommerce', 'newsletter' ] }
 			/>
-			<Categories onSelect={ ( c ) => doSearch( c.tags?.[ 0 ] || '' ) } />
+			<Categories onSelect={ ( c ) => doSearch( search, { tag: c.tags?.join( ',' ) || '' } ) } />
```

Comment:
I'm thinking this shouldn't be using doSearch, instead we should be linking to the `/plugins/:category` route in the dropdown, if a search term is present we can link to `/plugins/:category?s=:search` instead.